src/Components/EventComments.js

```diff
@@ -4,7 +4,7 @@ import { graphql } from "react-apollo";
 import moment from 'moment';
 
 import QueryGetEvent from "../GraphQL/QueryGetEvent";
-import SubsriptionEventComments from "../GraphQL/SubsriptionEventComments";
+import SubscriptionEventComments from "../GraphQL/SubscriptionEventComments";
```

Comment:
@kayaman Thanks for doing this! I do see that the file name itself is called [SubsriptionEventComments](https://github.com/aws-samples/aws-mobile-appsync-events-starter-react/blob/master/src/GraphQL/SubsriptionEventComments.js). Can you please update the name of the file as well within this PR? That way this doesn't break the sample